README.RDAP.md

```diff
@@ -34,45 +44,83 @@ Entity Primary Key can match multiple objects
 ---------------------------------------------
 If an entity primary key matches more than one object, a 500 Internal Server Error is returned.
 
+This can not be easily fixed because the same key can be used in multiple different object types: mntner and
+person/role. So it is not clear which object must be returned for a single object request.
+
 For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
 
 Related Contact information is Filtered
 ---------------------------------------
-Any related contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail" 
+Some related contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail"
 and "notify" values are not included. This was done to avoid blocking clients for inadvertently querying excessively for personal data.
 
-A workaround is to query for each entity separately using the contact's nic-hdl, and the unfiltered information is returned (although a limit for personal data does apply).
+For /entity/ request "e-mail" will be enabled and the clients requesting those service will need to comply with the daily limit
+according to the AUP: https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use
+-policy.
+
+For the rest of request either "notify" and "e-mail" is filtered.
 
 Abuse Contact information
 --------------------------
-Abuse contact information is not filtered. However, this attribute's type is not "home" or "work" as the RFC specifies. 
-The type of this attribute is "abuse". 
-
-This decision was taken to differentiate between a normal e-mail and abuse e-mail. Nowadays, with the rest of contact 
-entities e-mail attributes filtered (and redacted), this change can not be done because it is a breaking change.
+Abuse contact information is not filtered because it is not considered personal information. However, this attribute's
+`type` non-conforming to the [RDAP spec](https://bitbucket.org/nroecg/nro-rdap-profile/raw/v1/nro-rdap-profile.txt)
+section 5.1.1, is not "home" or "work" as the RFC specifies. The `type` of this attribute is "abuse".
+
+In the previous paragraph `type` is considered as an element of the Jcard
```

Comment:
Move that sentence into the previous paragraph as it's related to it, and then have a new paragraph just for the example